examples/jobs/passjob_html_browser.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+import sys
+
+from avocado.core.job import Job
+from avocado.core.suite import TestSuite
+
+config = {'run.references': ['examples/tests/passtest.py:PassTest.test'],
+          'job.run.result.html.enabled': 'on',
+          'job.run.result.html.open_browser': True}
```

Comment:
ohhh and here are again. :/

Reply:
> ohhh and here are again. :/

Right, but this doesn't get run on the selftests! :)